skeleton/template.yaml

```diff
@@ -86,6 +86,31 @@ spec:
                   type: string
                   ui:help: "The name of the model deployed on the model server you would like to use."
                 # SED_LLM_SERVER_END
+                includeModelEndpointSecret:
+                  title: Is bearer authentication required?
+                  type: boolean
+                  default: false
+                  ui:help: Create a secret containing the authentication bearer in the preffered targeted Namespace first.
```

Comment:
```suggestion
                  ui:help: Create a secret containing the authentication bearer in the preferred targeted Namespace first.
```

Reply:
Do we want to capitalize Secret? It is a Kubernetes resource after all like Namespace..

Reply:
For me makes sense. @MariaLeonova wdyt?

Reply:
I think the only unresolved item of this PR is if we'd like to use `Secret` instead of `secret`. Once this is resolved I'll merge it :)

cc @MariaLeonova @maysunfaisal

Reply:
@thepetk I agree with @maysunfaisal, let's capitalize to be consistent with other Kubernetes resources.
https://kubernetes.io/docs/concepts/configuration/secret/.

Reply:
Addressed here: [86b2275](https://github.com/redhat-ai-dev/ai-lab-template/pull/40/commits/86b227512bbab1040e3b2b01e138c341f0939e1d)